### DISCUSSION TITLE
Domain now.sh to vercel.app breaking CI

### DISCUSSION THREAD

**Discussion by Vadorequest** [vercel/vercel#4640] [Help] [answered]
Today, we encountered `##[error]Process completed with exit code 1.` when running `now alias` from our Github Actions workflow.

https://github.com/UnlyEd/next-right-now/runs/766101048

It's been fixed by https://github.com/UnlyEd/next-right-now/pull/109/commits/faadf9695bf791a87b492e9f0cd70deb52317cb5, basically replacing our alias from `now.sh` to `vercel.app`. ([see working action](https://github.com/UnlyEd/next-right-now/runs/766183371))

But, it was working fine this morning, nothing is mentioned in https://www.notion.so/Change-Timeline-fab7622780694753a92c0a396cea7eb6 and I don't understand why the sudden fail.

---

Also, I tried to create aliases from my own computer (e.g: `now alias nrn-v2-mst-aptd-at-lcz-sty-c1-93kav5g8k.vercel.app nrn-xxxkjkkskndsqkdsq.now.sh`), and it's failing with `Error! You don't have access to the domain nrn-xxxkjkkskndsqkdsq.now.sh under unly-oss.`. But I checked using `now switch` and I'm on the `unly-oss` team, as I thought.

```
 ✔  now alias nrn-v2-mst-aptd-at-lcz-sty-c1-93kav5g8k.vercel.app nrn-xxxkjkkskndsqkdsq.now.sh    
> UPDATE AVAILABLE Run `yarn add now@latest` to install Now CLI 19.1.0
> Changelog: https://github.com/zeit/now/releases/tag/now@19.1.0
Now CLI 17.1.1
> Assigning alias nrn-xxxkjkkskndsqkdsq.now.sh to deployment nrn-v2-mst-aptd-at-lcz-sty-c1-93kav5g8k.vercel.app
Error! You don't have access to the domain nrn-xxxkjkkskndsqkdsq.now.sh under unly-oss.
```

**Edit**: I had to upgrade the `now` CLI, I was using `17.1.1` locally, upgrading to `19.1.0` fixed it.

```
 ✔  now alias nrn-v2-mst-aptd-at-lcz-sty-c1-93kav5g8k.vercel.app nrn-xxxkjkkskndsqkdsq.now.sh
Now CLI 19.1.0
> Assigning alias nrn-xxxkjkkskndsqkdsq.now.sh to deployment nrn-v2-mst-aptd-at-lcz-sty-c1-93kav5g8k.vercel.app
> Success! https://nrn-xxxkjkkskndsqkdsq.now.sh now points to https://nrn-v2-mst-aptd-at-lcz-sty-c1-93kav5g8k.vercel.app [3s]

```

The error message is quite misleading.

**Comment by mglagola** [comment, marked as answer]
👋  @Vadorequest We pushed a fix to our api around 5:15pm UTC. Let us know if you're still running into the issue!

**Reply by Vadorequest** [reply]
Thanks! I started running into this issue around 18:05 CEST

It was still an issue one hour ago, so I'm not quite sure the issue is resolved.
https://github.com/UnlyEd/next-right-now/runs/766411712

The above action failed due to using `now alias` with `now.sh` domains.

I fixed it in that branch by changing `now.sh` to `vercel.app` as you can see at https://github.com/UnlyEd/next-right-now/pull/112

**Reply by mglagola** [reply]
@Vadorequest 

```
ZEIT_DEPLOYMENT_URL=`echo $ZEIT_DEPLOYMENT_OUTPUT | egrep -o 'https?://[^ ]+.now.sh'`
```

Based on the github action logs, im only seeing `https://<...>.vercel.app` being logged which may be why `now alias` is failing (`ZEIT_DEPLOYMENT_URL` is empty perhaps?).

**Reply by Vadorequest** [reply]
Yes, exactly. The Vercel platform behaviour's changed without priori warning today. Before, the alias was using .now.sh, but now it's using .vercel.app.

It's fixed for me, I changed the alias, but I believe others will run into this soon.
